precode.go

```diff
@@ -36,6 +36,7 @@ func Worker(in <-chan int64, out chan<- int64) {
 		out <- v
 		time.Sleep(1 * time.Millisecond)
 	}
+	defer close(out)
```

Comment:
Дефер нужно в начале ставить, а в конце можно просто закрыть
```suggestion
	close(out)
```